back-end/calender/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from basemodel.models import BaseModel
+
+
+# Create your models here
+
+class Calender(BaseModel):
+    name = models.TextField(blank=True,null=True,max_length=50)
```

Comment:
What exactly is a Calendar's name?

Reply:
Calendar Name is something like Placement 2022-2023. This could also be found from year details and is_internship field.

Reply:
Resolving conversation